src/Compilers/Core/Portable/Symbols/SymbolKind.cs

```diff
@@ -106,5 +106,10 @@ public enum SymbolKind
         /// Symbol represents a value that is discarded, e.g. in M(out _)
         /// </summary>
         Discard = 19,
+
+        /// <summary>
+        /// Symbol represents a member of an enum
+        /// </summary>
+        EnumMember = 20,
```

Comment:
we very likely would *not* want to change the low level compiler API here.  It would have a huge impact on all existing code that uses these API. :)